kivy/core/window/window_egl_rpi.py

```diff
@@ -26,6 +26,9 @@ class WindowEglRpi(WindowBase):
 
     _rpi_dispmanx_id = int(environ.get("KIVY_BCM_DISPMANX_ID", "0"))
     _rpi_dispmanx_layer = int(environ.get("KIVY_BCM_DISPMANX_LAYER", "0"))
+    _rpi_dispmanx_overlay = int(environ.get("KIVY_BCM_DISPMANX_OVERLAY", "0"))
```

Comment:
This needs to be documented at https://github.com/kivy/kivy/blob/715417e389b3d13115811cfccaf8b408963739e0/doc/sources/guide/environment.rst.